Dockerfile.devkit

```diff
@@ -0,0 +1,31 @@
+FROM debian:buster
```

Comment:
Dockerfile.devkit is the image that the tests run in and is the development image, Dockerfile.dcos-e2e is the image that just runs `minidcos` to setup the cluster. Eventually I'd like to see dcos-e2e publish a docker image so that installation is unnecessary. This is an attempt at prototyping out what that would look like. With this, the only requirements for development and setting up a local cluster is having docker and curl installed, everything else is run in a container.

Reply:
> Eventually I'd like to see dcos-e2e publish a docker image so that installation is unnecessary.

@jkoelker I'm very impressed by this. @mhrabovcin , @jgehrcke and I discussed doing this in the E2E repo some time ago.

Perhaps you could make an issue for upstreaming this to DC/OS E2E and our release process could publish a Docker image?

Reply:
@adamtheturtle sure, I'll put together a PR with appropriate docs on how to use.

Reply:
@jkoelker - ok that sounds good, thank you. Maybe the first step could be an issue created now-ish?